diagnostic.md

```diff
@@ -3,36 +3,48 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+faster loading speeds once page loads.
+routes make more sense.
+allows for quicker development--when understood.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+the handlebars templating library. It is handlebars.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+route.
+model.
+component.
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+the template is what is generated with ember.
+The component is a particular part of a route.
```

Comment:
Not quite. A component is a reusable piece of code that has data and functionality linked to a template.
A template is just the part that receives data and displays it.
0/1